pkg/core/localize/locales/en/cmd/generate_config.en.toml

```diff
@@ -43,4 +43,8 @@ one = 'file {{.FilePath}} already exists. Use --overwrite to overwrite the file,
 one='No services available to generate configurations'
 
 [generate.log.info.credentialsSaved]
-one='Configurations successfully saved to "{{.FilePath}}"'
\ No newline at end of file
+one='''
+Configurations successfully saved to "{{.FilePath}}"
+
+You can now create new service accounts or use existing ones to connect to the service(s)
```

Comment:
As user I'm wondering how.
Can we be more specific.

Run rhoas service-account create etc.